src/Compute/Compute.sln

```diff
@@ -14,6 +14,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Accounts", "..\Accounts\Acc
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "KeyVault", "..\KeyVault\KeyVault\KeyVault.csproj", "{9FFC40CC-A341-4D0C-A25D-DC6B78EF6C94}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ManagedServiceIdentity", "..\ManagedServiceIdentity\ManagedServiceIdentity.csproj", "{228EB071-FA04-43B3-95F9-7D76DBF0B850}"
```

Comment:
I added the reference to `MananagedServiceIdentity.csproj` but it's reverted in #11426 .
This ref is necessary because Compute's test cases depend on `Az.ManagedServiceIdentity`, and we want that every change in `Az.ManagedServiceIdentity` would trigger Compute's cases, making sure they work.  cc @hyonholee